example/lib/navigation/navigation_event.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:blemulator_example/model/ble_peripheral.dart';
+import 'package:equatable/equatable.dart';
+
+abstract class NavigationEvent extends Equatable {
+  const NavigationEvent();
+
+  @override
+  List<Object> get props => [];
+}
+
+class NavigatorPop extends NavigationEvent {}
```

Comment:
If the other is `NavigateTo`, then shouldn't this one be Back or Return or something?

Does `Navigator` prefix has any merits? It's inheriting from NavigationEvent already and exists in navigation_event.dart, so all the information is there.

Reply:
Indeed, the prefix was unnecessary. Renamed to just `Pop` (to keep in line with Flutter's `Navigator.pop`).